OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -97,7 +97,7 @@ public interface ITickRender { void TickRender(WorldRenderer wr, Actor self); }
 	public interface IRender { IEnumerable<IRenderable> Render(Actor self, WorldRenderer wr); }
 
 	public interface IAutoSelectionSizeInfo : ITraitInfoInterface { }
-	public interface IAutoSelectionSize { int2 SelectionSize(Actor self); }
+	public interface IAutoSelectionSize { int2 SelectionSize(Actor self); int2 SelectionOffset(Actor self); }
```

Comment:
Style nit: would be more readable to put the bracket content on newlines, now that it contains more than one.